Panel updates (#81)
djangocon/2024.djangocon.us: src/_content/schedule/talks/panel-discussion-django-communities.md

track: t0
---

Lacey Henschel will moderate a panel discussion on the history and future of communities within the Django community.
We'll explore how mentoring, community involvement, and support networks are important in keeping Django resilient. By highlighting the work of groups like Django Girls, PyLadies, Djangonaut Space, and Black Python Devs, we’ll see how these initiatives not only foster personal growth but also play a vital role in keeping the Django community strong and sustainable for the long term.

We’ll discuss:

- Panelists' unique journeys in the Django community
- The role of mentorship, sponsorship, and advocacy in these communities and in Django's future
- Successes and challenges in building supportive, sustainable communities
- How the Django community can keep investing in long-term success

Our panelists include:
- **Lacey Henschel**, moderator, former Django Girls contributor and organizer
- **Keanya Phelps**, Django Girls organizer and participant, mentor in Djangonaut Space
- **Afi Maame Dufie**, involved in Django Girls, PyLadies, and Black Python Devs
- **Sarah Boyce**, Django Fellow, co-founder and mentor in Djangonaut Space

> - **Carlton Gibson**, former Django Fellow, mentor in Djangonaut Space, contributor to Google Summer of Code and Django Girls

Reviewer: The GSoC bit didn't feel right. (I never was a GSoC _contributor_...) Maybe: 

```suggestion
- **Carlton Gibson**, former Django Fellow, mentor in Djangonaut Space, mentor and coordinator of Django's efforts in Google Summer of Code and Django Girls coach and advisory board member.
```

> - **Carlton Gibson**, former Django Fellow, mentor in Djangonaut Space, mentor and coordinator of Django's efforts in Google Summer of Code and Django Girls coach and advisory board member.

Reviewer 2: ```suggestion
- **Carlton Gibson**, former Django Fellow, mentor in Djangonaut Space, mentor and coordinator of Django's efforts in Google Summer of Code, and Django Girls coach and advisory board member.
```

Just adding in an Oxford comma for consistency

> - **Ron Maravanyika**, co-founder of ZimboPy, current chair for PyCon Zimbabwe, and Django Girls organizer

Reviewer 2: ```suggestion
- **Velda Kiara**,  DEFNA director, former [AnitaB.org](http://anitab.org/) Kenya Community Lead, former Open Source Community Africa Nairobi organizer and contributor, participant in Djangonaut Space, and current contributor and leader of Black Python Devs
```
Replacing Ron with Velda on the panel
